README.md

```diff
@@ -72,25 +72,23 @@ Depending on backend type, you might need Linux `sudo` to be used when running P
 ### Bluetooth Connection Options
 There is an optional parameter for `MoveHub` class constructor, accepting instance of `Connection` object. By default, it will try to use whatever `get_connection_auto()` returns. You have several options to manually control that:
 
-- use `pylgbst.get_connection_auto()` to attempt backend auto-choice, autodetect uses 
-- use `BlueGigaConnection()` - if you use BlueGiga Adapter (`pygatt` library prerequisite)
-- use `GattConnection()` - if you use Gatt Backend on Linux (`gatt` library prerequisite)
-- use `GattoolConnection()` - if you use GattTool Backend on Linux (`pygatt` library prerequisite)
-- use `GattLibConnection()` - if you use GattLib Backend on Linux (`gattlib` library prerequisite)
-- use `BluepyConnection()` - if you use Bluepy backend on Linux/Raspbian (`bluepy` library prerequisite)
-- use `BleakConnection()` - if you use Bleak backend (`bleak` library prerequisite)
+- use `pylgbst.get_connection_auto()` to attempt backend auto-detection 
+- use `pylgbst.get_connection_bluegiga()` - if you use BlueGiga Adapter (`pygatt` library prerequisite)
```

Comment:
IMO repeating the prefix garbages the documentation. Developer would be able to specify imports correctly. Using this lib implies basic Python knowledge.

Reply:
see my other comment for an idea how to clean this whole thing up